### DISCUSSION TITLE
Iptv-Brasil-2023

### DISCUSSION THREAD

**Discussion by mrfall22** [Ramys/Iptv-Brasil-2026#88] [Em Geral]
todos os dias de manha, todas as series e filmes param de funcionar, tem alguma coisa que possa ser feito? tem como eu ajudar com algo?

**Comment by Ramys** [comment]
Atualizei as listas..
Teste.

**Comment by mrfall22** [comment]
Obrigado Ramys, tudo certo um abração!
